app/views/spree/admin/shared/_head.html.erb

```diff
@@ -20,6 +20,9 @@
   <%== "var OAUTH_TOKEN = '#{admin_oauth_token}';" if defined?(admin_oauth_token) %>
   <%== "var ADDRESS_FIELDS = #{Spree::Address::ADDRESS_FIELDS}" %>
 </script>
+
+<%= javascript_include_tag "application", data: { turbo_track: "reload" } if Rails.env.test? %>
```

Comment:
The way Spree/Dashboard is currently setup, if you are going to use Turbo for the admin UI then you are going to need this for more than just the test enviroment.

Or, the Spree user manually overides the `_head.html.erb` and adds in thir own javascript that imports hotwire/turbo.